packages/create-react-app/index.js

```diff
@@ -264,6 +270,28 @@ function checkAppName(appName) {
   }
 }
 
+function moveReactScriptsToDev(packageName) {
+  var packagePath = path.join(process.cwd(), 'package.json');
+  if (!pathExists.sync(packagePath)) {
+    return;
+  }
+
+  var packageJson = require(packagePath);
+  if (typeof packageJson.dependencies === 'undefined') {
+    return;
+  }
+
+  var packageVersion = packageJson.dependencies[packageName]
+
+  if (typeof packageVersion !== 'undefined') {
```

Comment:
Same as the other check, if this is `undefined` it's an error and the script should exit.